src/OrchardCore.Modules/OrchardCore.Deployment/wwwroot/Scripts/steporder.js

```diff
@@ -22,7 +22,7 @@ function updateStepOrders(oldIndex, newIndex) {
 }
 $(function () {
   var sortable = document.getElementById("stepOrder");
-  if (!sortable) {
+  if (sortable) {
```

Comment:
why is this change? I think it is correct to be !sortable

Reply:
This PR https://github.com/OrchardCMS/OrchardCore/pull/15306 changed the files by hand instead of  using gulp and different changes (and no changes in the min file) were merged.

The thing you are seeing here is just the result after running gulp correctly such that what is in the source file is reflected correctly in the generated ones.

Reply:
I see it. I'll fix it

Reply:
Fix what? I don't think anything needs to get fixed, I believe the condition is correct.

Reply:
the condition is wrong. Fixed here https://github.com/OrchardCMS/OrchardCore/pull/15387